sdk/nodejs/ec2/defaultNetworkAcl.ts

```diff
@@ -30,65 +30,6 @@ import * as utilities from "../utilities";
  * For more information about Network ACLs, see the AWS Documentation on
  * [Network ACLs][aws-network-acls].
  * 
- * ## Basic Example Usage, with default rules
```

Comment:
Did the upstream TF lose these examples? cc @joeduffy

Reply:
This particular one became invalid: it was edited to stop specifying "0.0.0.0/0" as the CIDR block, and instead has a comment saying "# set a CIDR block here", which causes parsing in `tf2pulumi` to fail with:

```
error: creating tree module: Error parsing main.tf: At 13:5: Unknown token: 13:5 IDENT from_port
```